src/vs/base/browser/ui/icons/iconSelectBox.ts

```diff
@@ -203,11 +221,22 @@ export class IconSelectBox extends Disposable {
 	}
 
 	focusNextRow(): void {
-		this.focusIcon((this.focusedItemIndex + this.numberOfElementsPerRow) % this.renderedIcons.length);
+		let nextRowIndex = this.focusedItemIndex + this.numberOfElementsPerRow;
+		if (nextRowIndex >= this.renderedIcons.length) {
+			nextRowIndex = (nextRowIndex % this.numberOfElementsPerRow) + 1;
```

Comment:
```suggestion
			nextRowIndex = (nextRowIndex + 1) % this.numberOfElementsPerRow;
```
Then, I don't think the next line is needed

Reply:
Yes. Thanks